stock_picking_filter_lot/readme/CONTRIBUTORS.rst

```diff
@@ -3,3 +3,4 @@
 * Alan Ramos <alan.ramos@jarsa.com.mx> (www.jarsa.com.mx)
 * Tharathip Chaweewongphan <tharathipc@ecosoft.co.th> (www.ecosoft.co.th)
 * Jesus Alcala <jesus.alcala@jarsa.com.mx> (www.jarsa.com.mx)
+* Le Filament (le-filament.com)
```

Comment:
Here is the place for individuals

Reply:
Hello @remi-filament Do you want to provide your informations or shall I remove it?

Reply:
Thanks @leemannd, it comes from my mistake on v14 PR. If you do not mind, you can set :
Remi Cazenave \<remi-filament\> (<https://le-filament.com>)

Thanks !

Reply:
PR updated. @remi-filament your review is welcome